voicevox_engine/model.py

```diff
@@ -131,6 +131,7 @@ class StyleInfo(BaseModel):
 
     id: int = Field(title="スタイルID")
     icon: str = Field(title="当該スタイルのアイコンをbase64エンコードしたもの")
+    style_portrait: Optional[str] = Field(title="当該スタイルのportrait.pngをbase64エンコードしたもの")
```

Comment:
```suggestion
    portrait: Optional[str] = Field(title="当該スタイルのportrait.pngをbase64エンコードしたもの")
```

`run.py`の部分を変更する場合この部分も変更する必要がある